src/components/DiverInfomation/DiverInfo.spec.js

```diff
@@ -1,18 +1,17 @@
+import { userEvent } from '@storybook/testing-library';
```

Comment:
Do you want to use the storybook testing library or was it the autocompletion? 😉